### DISCUSSION TITLE
Can I remove the extra wrapper in <div>?

### DISCUSSION THREAD

**Discussion by iakovalgaev** [table-library/react-table-library#74] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/61377069/184844614-d87c0d2b-4f86-4fe0-beea-94ad85649b86.png)

**Comment by iakovalgaev** [comment]
I still need it

**Comment by rwieruch** [comment, marked as answer]
At the moment the div HTML element is there for structural reasons and not easily removable. However, can you make your scenario happen by styling the div to your needs?

```
const theme = useTheme({
  BaseCell: `
    div {
      color: blue;
    }
  `,
});
```
